src/AstroImages.jl

```diff
@@ -92,17 +106,19 @@ function AstroImage(file::String)
 end
 
 # Lazily render the image as a Matrix{Color}, upon request.
-function render(img::AstroImage{T,C}) where {T,C}
-    imgmin, imgmax = extrema(img.data)
+function render(img::AstroImage{T,C,N}, header_number = 1) where {T,C,N}
```

Comment:
I think this method is good as it is now for `N=1`.  I'd define a different method for `N=3`

Reply:
Do you mean, I should use dispatch and write them separately as two functions?

Reply:
Yes, but given your other message about support for RGB colour maybe you don't want to add a method for `N=3` now

Reply:
Yeah, that's why I was a bit confused about your comment as currently, this does the thing as I want it to.